apps/docs/pages/docs/message-signing/standard-messages.en-US.mdx

```diff
@@ -5,12 +5,19 @@ description:
 ---
 
 import * as Integrations from 'components/integrations-tabs';
+import { WithAuth } from 'components/with-auth';
 import Callout from 'nextra-theme-docs/callout';
 
 # Standard message signing
 
 <Integrations.Tabs />
 
+<WithAuth>
+
+This will only show up if you have authenticated!
+
+</WithAuth>
```

Comment:
@Aslemammad here is an example of how we can gate the content unless they are authenticated